README.md

```diff
@@ -106,6 +106,12 @@ Create an expiring cookie, valid to the path of the current page:
 Cookies.set('name', 'value', { expires: 7, path: '' })
 ```
 
+Create a cookie with SameSite attributes:
```

Comment:
I think it should say “... SameSite attribute” (no plural, there’s only one such attribute).

Reply:
Ah, it refers to the additional `secure` flag I guess.